Update GitHub setup docs for IDP Homepage (#38755)
DataDog/documentation: content/en/internal_developer_portal/homepage.md

The Homepage aggregates data from your Datadog integrations. To populate the task sections, configure the following integrations and webhooks before using the Homepage:

> - **GitHub**: An administrator installs and and configures the GitHub integration, and sets up a GitHub webhook so that repository events (such as pull request activity) reach Datadog in real time. The **GitHub PRs** section requires this webhook to display current pull request data.  Each user also signs in with their own GitHub account to load the pull requests relevant to them. For setup steps, see [GitHub][1].

Reviewer: ```suggestion
- **GitHub**: An administrator installs and configures the GitHub integration, and sets up a GitHub webhook so that repository events (such as pull request activity) reach Datadog in real time. The **GitHub PRs** section requires this webhook to display current pull request data. Each user also signs in with their own GitHub account to load the pull requests relevant to them. For setup steps, see [GitHub][1].
```

- **Status** (Open / Draft / Merged)  
- **Assignee / Reviewer**  

> Two setup steps control what appears in this section, in order: an administrator first connects the GitHub integration for the organization, then each user signs in with their own GitHub account. After you authorize access, the section loads your pull requests, grouped by status. 

Reviewer: ```suggestion
This section requires two setup steps, in order: an administrator connects the GitHub integration for the organization, and each user signs in with their own GitHub account. After you authorize access, the section loads your pull requests, grouped by status. 
```
